post-processor/googlecompute-export/post-processor.go

```diff
@@ -68,6 +68,12 @@ type Config struct {
 	Zone                string `mapstructure:"zone"`
 	IAP                 bool   `mapstructure-to-hcl2:",skip"`
 	ServiceAccountEmail string `mapstructure:"service_account_email"`
+	//If true, the export instance will not have an external IP. use_internal_ip must
+	//be true if this property is true.
+	OmitExternalIP bool `mapstructure:"omit_external_ip" required:"false"`
+	//If true, use the export instance's internal IP instead of its external IP
+	//during exporting.
+	UseInternalIP bool `mapstructure:"use_internal_ip" required:"false"`
```

Comment:
Looking at the usage of this attribute it would appear that we only need one attribute `use_internal_ip` as you can not have one be true without the other. Is that correct? 

```suggestion
	//By default the external IP used by the exporting machine have an external accessible IP.
	//If you need to suppress the use of an external IP for security reasons set this configuration to `true`
	// to force the export instance its internal IP instead of its external IP during exporting.
	UseInternalIP bool `mapstructure:"use_internal_ip" required:"false"`
```